src/main/java/spoon/support/compiler/jdt/CompilationUnitWrapper.java

```diff
@@ -43,30 +44,14 @@
 			return jdtCompiler.loadedContent.get(s);
```

Comment:
does this cache mean that one cannot modify - compile and then modify again and compile again?